### ISSUE TITLE
patch-agx7-2023-3-2.tar.gz is not in gzip format

### ISSUE THREAD

**Issue by calvinckhung** [OFS/ofs-n6001#11]
Hi, it seems the file is corrupted. You may want to double-check the integrity of each asset.

```
$ tar -zxvf patch-agx7-2023-3-3.tar.gz

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now

$ gunzip ./patch-agx7-2023-3-2.tar.gz 

gzip: ./patch-agx7-2023-3-2.tar.gz: not in gzip format
```

**Comment by cohensus** [comment]
Hi Calvin,
I noticed that you used the command tar -zxvf patch-agx7-2023-3**-3**. tar.gz , but the file name is different than what you have above.  The file name has a "-2" instead of "-3".  I just downloaded the asset and was able to untar it.  

[PROMPT]$ tar -xvf patch-agx7-2023-3-2.tar.gz
patch-agx7-2023-3/
patch-agx7-2023-3/quartus-0.0-0.02iofs-linux.run
patch-agx7-2023-3/quartus-23.3-0.13-linux.run
patch-agx7-2023-3/quartus-23.3-0.13-readme.txt
patch-agx7-2023-3/quartus-23.3-0.21-linux.run
patch-agx7-2023-3/quartus-23.3-0.21-readme.txt

Are you referring to this file or another?
